mlflow/langchain/__init__.py

```diff
@@ -905,23 +905,21 @@ def _load_model_from_local_fs(local_model_path, model_config_overrides=None):
         flavor_code_path = pyfunc_flavor_conf.get(
             MODEL_CODE_PATH, flavor_conf.get(MODEL_CODE_PATH, None)
         )
-        code_path = os.path.join(
+        model_code_path = os.path.join(
             local_model_path,
             os.path.basename(flavor_code_path),
         )
-
+        _add_code_from_conf_to_system_path(local_model_path, flavor_conf)
```

Comment:
It should be moved to front of L896 because you removed L924

Reply:
@harshilprajapati96 Could you also improve test to make it cover this case ? :)

Reply:
added test to cover the else condition!